buildSrc/src/main/groovy/maru.kotlin-common-conventions.gradle

```diff
@@ -14,6 +14,7 @@ dependencies {
   } because("We are using Teku's SafeFuture, an improved/extension of CompletableFuture.")
 
   implementation "com.michael-bull.kotlin-result:kotlin-result"
+  implementation "build.linea.internal:metrics"
```

Comment:
I wouldn't put it here, because I think that it won't be that many gradle modules that will expose any metrics